spec/spec_helper.rb

```diff
@@ -14,7 +14,6 @@
 require "action_controller"
 require "action_mailer"
 require "pry"
-require "webmock/rspec"
```

Comment:
Thank you for binning off Webmock! I 100% agree we should trust the client to do what it says it will